warn/scrapers/ga.py

```diff
@@ -11,111 +12,125 @@
 __tags__ = ["html"]
 
 
-def scrape(
-    data_dir: Path = utils.WARN_DATA_DIR,
-    cache_dir: Path = utils.WARN_CACHE_DIR,
-) -> Path:
-    """
-    Scrape data from Georgia.
+class NoticeListSource(URL):
```

Comment:
The reason for this source class is to fit in the cache writing. A better design would factor this out of this specific scraper and then you'd just rely on a `CachedSource`.